### DISCUSSION TITLE
Support for variants of a locale

### DISCUSSION THREAD

**Discussion by ocean90** [GlotPress/GlotPress-WP#1333] [Ideas]
Using this discussion as a place to document the various issues and pull requests related to variants support which was planned for 3.0 but was later reverted due to a number of issues which couldn't be resolved in time.

* https://github.com/GlotPress/GlotPress-WP/issues/226
   * https://github.com/GlotPress/GlotPress-WP/issues/747
   * https://github.com/GlotPress/GlotPress-WP/pull/1327
* https://github.com/GlotPress/GlotPress-WP/issues/1018
* https://github.com/GlotPress/GlotPress-WP/issues/1019
* https://github.com/GlotPress/GlotPress-WP/issues/1234
* https://github.com/GlotPress/GlotPress-WP/issues/1301
   * https://github.com/GlotPress/GlotPress-WP/issues/1302
* https://github.com/GlotPress/GlotPress-WP/issues/1303

If this gets picked up again, the above issues will be a good starting point.

**Comment by c-schmitz** [comment]
Will this ever be picked up again? We are maintaining 80+ translations and the feature to inherit translations into a variant from a core translation would be a huge benefit.
